.github/workflows/pr.yml

```diff
@@ -53,5 +60,17 @@ jobs:
       with:
         dotnet-version: 5.0.*
 
+    - name: Setup Node and 262 test suite
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
```

Comment:
Why not newer version with package lock files v2?

Reply:
I'm just copy-pasting here from the past, I can check to update this one

Reply:
Now using logic to download repository archive, so long package.json and whatnot.